lib/set.rb

```diff
@@ -62,6 +62,161 @@
 #
 # - Akinori MUSHA <<knu@iDaemons.org>> (current maintainer)
 #
+# ## What's Here
+#
+# First, what's elsewhere. \Set includes the module
+# {Enumerable}[https://docs.ruby-lang.org/en/master/Enumerable.html],
```

Comment:
I'm not sure it's a good idea to add this link.  Set can be used with multiple Ruby versions, and it's not a good idea to directly link to documentation for a potentially different version (especially the master branch, which may have changes that no released version has).